source/backends/debian/debpkgindex.d

```diff
@@ -195,7 +200,7 @@ public:
             if (!name)
                 continue;
 
-            auto pkg = new DebPackage (name, ver, arch);
+            auto pkg = newPackage (name, ver, arch);
```

Comment:
We need to either make a `DebPackage` or an `UbuntuPackage`, so I added a factory method here. Let me know if you can think of a nicer way to do that.